src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1279,6 +1279,7 @@ namespace ReportSizingManager {
                         }
                     }
                 } else if (SizingType == CoolingWaterflowSizing) {
+                    CoilDesWaterDeltaT = DataWaterCoilSizCoolDeltaT;
```

Comment:
This is the only line I changed. Looks like clang format can't make up it's mind from one commit to the next.

Reply:
Must be some slight differences in how the clang format rules are applied? Curious - are you using the recent VS with built-in clang formatting, or the plugin with an earlier VS?

Reply:
I'm using Community 2017 at home (this commit) and Professional 2017 at work. I see Enable ClangFormat support under Tools/Options/Text Editor/CC++/Formatting (box checked). I can't say that I know how to set up clang formatting or even if I have to do something but it appears to be working.